File: src/main/java/org/minturtle/careersupport/common/utils/NanoIdGenerator.java
package org.minturtle.careersupport.common.utils;

import com.aventrix.jnanoid.jnanoid.NanoIdUtils;

import java.util.Random;

public abstract class NanoIdGenerator {

    public static String createNanoId(int idSize){
        Random random = new Random();

        return NanoIdUtils.randomNanoId(random, NanoIdUtils.DEFAULT_ALPHABET,idSize);

    }

    public static String createNanoId(){
        return createNanoId(10);
    }

}

File: src/main/kotlin/org/minturtle/careersupport/common/utils/NanoIdGenerator.kt
package org.minturtle.careersupport.common.utils

import com.aventrix.jnanoid.jnanoid.NanoIdUtils
import java.util.*

object NanoIdGenerator {

    fun createNanoId(idSize: Int = 10): String {
        val random = Random()
        return NanoIdUtils.randomNanoId(random, NanoIdUtils.DEFAULT_ALPHABET, idSize)
    }

}